common/djangoapps/student/views.py

```diff
@@ -1576,6 +1590,8 @@ def create_account_with_params(request, params):
                 user.email_user(subject, message, from_address)
         except Exception:  # pylint: disable=broad-except
             log.error(u'Unable to send activation email to user from "%s"', from_address, exc_info=True)
+    else:
+        registration.activate()
```

Comment:
@pmitros please don't ping team members directly - I've pinged Griff a few times for you. Please work with Griff to get this work into the team.